doc/source/data/api/from_other_data_libs.rst

```diff
@@ -0,0 +1,88 @@
+.. _api-guide-for-users-from-other-data-libs:
+
+API Guide for Users from Other Data Libraries
+=============================================
+
+Ray Datasets is a data loading and preprocessing library for ML. It shares certain
+similarities with other ETL data processing libraries, but also has its own focus.
+In this API guide, we will provide API mappings for users who come from those data
+libraries, so you can quickly map what you may already know to Ray Datasets APIs.
+
+Note:
+
+- This is meant to map APIs that perform comparable but not necessarily identical operations.
+  Please check the API reference for exact semantics and usage.
+- This list may not be exhaustive: Ray Datasets is not a traditional ETL data processing library, so not all data processing APIs can map to Datasets.
+  In addition, we try to focus on common APIs or APIs that are less obvious to see a connection.
+
+.. _api-guide-for-pandas-users:
+
+For Pandas Users
+----------------
+
+.. list-table:: Pandas DataFrame vs. Ray Datasets APIs
+   :header-rows: 1
+
+   * - Pandas DataFrame API
+     - Ray Datasets API
+   * - pandas.DataFrame.head()
+     - :meth:`ds.show() <ray.data.Dataset.show>` or :meth:`ds.take() <ray.data.Dataset.take>`
+   * - pandas.DataFrame.dtypes
+     - :meth:`ds.schema() <ray.data.Dataset.schema>`
+   * - len(df) or df.shape[0]
+     - :meth:`ds.count() <ray.data.Dataset.count>`
+   * - pandas.DataFrame.truncate()
+     - :meth:`ds.limit() <ray.data.Dataset.limit>`
+   * - pandas.DataFrame.iterrows()
+     - :meth:`ds.iterrows() <ray.data.Dataset.iter_rows>`
```

Comment:
Oops, missed this one: it should be `iter_rows`.

Reply:
Nice catch, I can add a fix now!